src/Plugin/Fuzzy/Payload.php

```diff
@@ -27,6 +27,7 @@
 final class Payload extends BasePayload {
 	const MAX_BOOST = 50;
 	const DECREASE_FACTOR = 1.44;
+	const ESCAPE_CHARS = '()[]<>!|*/-~^$@';
```

Comment:
I'm not sure about `>`

The list of chars to escape can be found here https://manual.manticoresearch.com/Searching/Full_text_matching/Escaping#Escaping-characters-in-query-string 

Pls re-check.